firmware/Makefile

```diff
@@ -61,7 +62,7 @@ endif
 
 # C++ specific options here (added to USE_OPT).
 ifeq ($(USE_CPPOPT),)
-  USE_CPPOPT = -std=c++17 -Wno-register -fno-rtti -fno-threadsafe-statics -fno-exceptions -fno-use-cxa-atexit -Winvalid-pch
```

Comment:
we should still include this option, right?

Reply:
No, gcc will just spam you even if it eventually finds a valid pch file.  Stupid design IMO.... so I guess we just will have to rely on developers complaining if their builds get slow...